aten/src/ATen/cpu/vec/vec256/vec256_convert.h

```diff
@@ -133,6 +133,50 @@ struct VecConvert<int32_t, 1, uint8_t, 1> {
   }
 };
 
+
+template <>
+struct VecConvert<int32_t, 1, float, 1> {
+  static inline VectorizedN<int32_t, 1> apply(
+      const VectorizedN<float, 1>& src) {
+    return  Vectorized<int32_t>(_mm256_cvttps_epi32(src[0]));
+  }
+};
+
+template <>
+struct VecConvert<float, 1, int32_t, 1> {
+  static inline VectorizedN<float, 1> apply(
+      const VectorizedN<int32_t, 1>& src) {
+    return  Vectorized<float>(_mm256_cvtepi32_ps(src[0]));
+  }
+};
+
+template <>
+struct VecConvert<int16_t, 1, uint8_t, 1> {
+  static inline VectorizedN<int16_t, 1> apply(
+      const VectorizedN<uint8_t, 1>& src) {
+    auto src128 = _mm256_castsi256_si128(src[0]);
+    return Vectorized<int16_t>(_mm256_cvtepu8_epi16(src128));
+  }
+};
+
+template <>
+struct VecConvert<float, 1, uint8_t, 1> {
+  static inline VectorizedN<float, 1> apply(
+      const VectorizedN<uint8_t, 1>& src) {
+    return VecConvert<float, 1, int32_t, 1>::apply(
+        VecConvert<int32_t, 1, uint8_t, 1>::apply(src));
+  }
+};
+
+template <>
+struct VecConvert<int8_t, 1, float, 1> {
```

Comment:
We should already have the specialization to convert float to uint8/int8

Reply:
Remove and only keep the original one.